src/index.js

```diff
@@ -7,13 +7,14 @@ import { EventEmitter } from 'events';
 
 import Storage from './storages/Storage';
 import JSONStorage from './storages/JSONStorage';
+import MongoDBStorage from './storages/MongoDBStorage';
 import SequelizeStorage from './storages/SequelizeStorage';
 
 /**
  * @class Umzug
  * @extends EventEmitter
  */
-module.exports = class Umzug extends EventEmitter {
+export default class Umzug extends EventEmitter {
```

Comment:
Double check that this is actually working? I don't want to broke the package again.

Reply:
You are right. So used to import/export that I totally forgot about classic require.
I will check it and will try to add a way to automatic test this. First idea I came up with is splitting up tests in unit tests (running against src) and integration tests (running against lib).